### PR TITLE
New guide: change hostname

### PR DESCRIPTION
**Pull request by krisdale**
New guide showing how to edit hostname on Clear Linux.

### CODE REVIEW

**Comment by krisdale** [comment]
@pixelgeek, @mvincerx, please review and provide any feedback needed. @tpleavitt optional to review.

File: source/clear-linux/guides/maintenance/hostname.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+   * *regression-test*
+   * *sally-test-box1*
+   * *az-bldg2-lab*
+
+
+Set your hostname
+*****************
+
+|CL| uses the :command:`hostnamectl` command to display and modify the 
+machine name. :command:`hostnamectl` is part of the **os-core** bundle.
```
Comment (mvincerx) [inline source/clear-linux/guides/maintenance/hostname.rst]: @krisdale. I still struggle with the wording of this. Can this sentence be removed, revised? What is the intent in saying that a Linux `command`, `hostnamectl`, is part of ... a bundle? Like we discussed, this is just a general Linux command.

**Comment by krisdale** [comment]
@pixelgeek please give approval on this PR, and we'll get it merged. Latest clarification re os-core bundle is included. Thanks!

**Comment by mvincerx** [comment]
@krisdale.  Revisions look great.  I appreciate additional explanation of how Clear Linux organizes its bundles.  Just a minor formatting error. Lines 34-35 and 57 need to moved one space to the right to render the commands correctly.  Think of it this way: More green squares!:)

**Comment by mvincerx** [comment]
Q3 objective met. Merged. @pixelgeek, please review, and if additional changes, request new PR.

**Pull request merged**